includes/API/FBE/Configuration/Read/Response.php

```diff
@@ -40,5 +40,58 @@ public function get_messenger_configuration() {
 		return $configuration;
 	}
 
+	/**
+	 * Gets feed configuration settings.
+	 *
+	 * @since x.x.x
+	 *
+	 * @return null|API\FBE\Configuration\CatalogFeedSchedule
+	 */
+	public function get_catalog_feed_configuration() {
+
+		$configuration = null;
+
+		if ( ! empty( $this->response_data->catalog_feed_scheduled ) && is_object( $this->response_data->catalog_feed_scheduled ) ) {
```

Comment:
In my testing, the `catalog_feed_schedule` is `false` despite the fact that I have a valid and active feed configuration.